src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/Formatting/Rules/NewLineUserSettingFormattingRule.cs

```diff
@@ -537,6 +562,7 @@ public override int GetHashCode()
                 hashCode = (hashCode << 1) + (NewLinesForBracesInProperties ? 1 : 0);
                 hashCode = (hashCode << 1) + (NewLinesForBracesInMethods ? 1 : 0);
                 hashCode = (hashCode << 1) + (NewLinesForBracesInAccessors ? 1 : 0);
+                hashCode = (hashCode << 1) + (NewLinesForBracesInPropertyPatternClauses ? 1 : 0);
                 hashCode = (hashCode << 1) + (NewLinesForBracesInAnonymousMethods ? 1 : 0);
                 hashCode = (hashCode << 1) + (NewLinesForBracesInLambdaExpressionBody ? 1 : 0);
                 hashCode = (hashCode << 1) + (NewLinesForBracesInControlBlocks ? 1 : 0);
```

Comment:
I'm on favor to convert this to a record struct once the feature is shipped in C# 10 🎉